SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -130,27 +130,28 @@ bool USpatialNetDriver::InitBase(bool bInitAsClient, FNetworkNotify* InNotify, c
 	PlayInEditorID = GPlayInEditorID;
 
 	// If we're launching in PIE then ensure there is a deployment running before connecting.
-	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
-	FLocalDeploymentManager* LocalDeploymentManager = GDKServices.GetLocalDeploymentManager();
+	if (FSpatialGDKServicesModule* GDKServices = FModuleManager::GetModulePtr<FSpatialGDKServicesModule>("SpatialGDKServices")) {
```

Comment:
Reason for adding this check? Seems if the module isn't loaded all bets are off.

Reply:
It's possible that you're running a local worker outside PIE with `UE4Editor.exe Project.uproject -server`, in which case the editor modules wouldn't be loaded and you'd be required to start a runtime externally, and it wouldn't make sense to attempt to do so from UE code as this code does.

It would probably be best to check if we're in PIE here, and crash if the module isn't loaded _and_ we're in PIE.

Reply:
Good point